server/src/main/java/org/elasticsearch/indices/recovery/RecoverySourceHandler.java

```diff
@@ -404,15 +415,17 @@ void phase1(IndexCommit snapshot, long globalCheckpoint, IntSupplier translogOps
                     phase1FileNames.size(), new ByteSizeValue(totalSizeInBytes),
                     phase1ExistingFileNames.size(), new ByteSizeValue(existingTotalSizeInBytes));
                 final StepListener<Void> sendFileInfoStep = new StepListener<>();
+                final StepListener<Void> sendFileChunkStep = new StepListener<>();
```

Comment:
`sendFilesStep` instead matches the method name. The "chunk" part I would regard an implementation detail at this level?